src/chai/ExecutionSpaces.hpp

```diff
@@ -43,6 +44,31 @@ enum ExecutionSpace {
   ,PINNED
 #endif
 };
+static std::vector<char*> PrintExecSpace = { 
```

Comment:
Is this for debugging? If not, you need to add `#include <vector>` to this file.

Reply:
For debugging, has been removed.